src/games/strategy/engine/framework/startup/ui/editors/MicroWebPosterEditor.java

```diff
@@ -163,10 +163,11 @@ private void initGame() {
     window.setLayout(new GridBagLayout());
     window.getContentPane().add(new JLabel("Select Players For Each Nation:"), new GridBagConstraints(0, 0, 2, 1, 0, 0,
         GridBagConstraints.CENTER, GridBagConstraints.NONE, new Insets(20, 20, 20, 20), 0, 0));
-    final JComboBox[] comboBoxes = new JComboBox[m_parties.length];
+    @SuppressWarnings("unchecked")
```

Comment:
No need for the @SuppressWarnings, the TODO alone is fine, in which case we are getting rid of the compiler warning. Easier to find the compiler warning than it is a todo.